projects/plugins/jetpack/extensions/blocks/premium-content/premium-content.php

```diff
@@ -151,10 +151,22 @@ function stripe_nudge( $checkout_url, $description, $button_text ) {
  * @return void
  */
 function add_paid_content_post_meta( int $post_id, WP_Post $post ) {
+	if ( $post->post_type !== 'post' ) {
```

Comment:
Hi! Would it make sense to track paid content block with the meta in all post types, particularly _pages_, since the block itself has that flexibility? Or the limit is just for AI-data reasons?

Reply:
@millerf Do you know the answer to this? I know the previous functionality was limited to posts, but I also didn't think about having paid content on pages.

Reply:
That's a good point. Let's allow [pages](https://developer.wordpress.org/themes/basics/post-types/#default-post-types) as well.

```suggestion
	if ( $post->post_type !== 'post' && $post->post_type !== 'page' ) {
```